api/v1alpha1/linodemachine_types.go

```diff
@@ -25,17 +26,59 @@ import (
 
 // LinodeMachineSpec defines the desired state of LinodeMachine
 type LinodeMachineSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	Region          string                                 `json:"region"`
+	Type            string                                 `json:"type"`
+	Label           string                                 `json:"label,omitempty"`
+	Group           string                                 `json:"group,omitempty"`
```

Comment:
What would this "Group" property mean for a Linode?

Reply:
I don't know @avestuk , this part of the code is a copy of https://github.com/linode/linodego/blob/1c6ad41c9ec16f2d1e583da338dc00d407629b22/instances.go#L113

Reply:
Ah ok - The docs say it's deprecated so maybe drop it? 

https://www.linode.com/docs/api/linode-instances/#linode-create

Reply:
@avestuk we should drop it, but we have a dynamic converter here: https://github.com/linode/cluster-api-provider-linode/pull/36/files#diff-8daab1045cecf480419b0a31c2f8558e51197d31a7be910386ef4e420db0f7a4R314 which in my best understanding is able to handle this problem, I just don't want to ensure :) Currently we have a simple test to verify integrity between the two objects: https://github.com/linode/cluster-api-provider-linode/pull/36/files#diff-a78247066969ae5bb22df9e10e6a41c5435d78a1ac24264c31a69df60ec8ff90R13 WDYT?

Reply:
Ah right its for the lindego client to create the machine. Yeah fair enough then leave it as is :D